2345/3/models/comment_parser.rb

```diff
@@ -0,0 +1,13 @@
+module CommentParser
```

Comment:
Looks like it should have the same structure as ArticleParser
Do you have some reason to use module in this case?

I would do smth like this:

```
class CommentParser
  def initialize(comment_data)
  end
  
  def votes
  end
  
  def author
  end

  def text
  end
end

```